templates/terraform/environments/prod/user_access.tf

```diff
@@ -110,4 +184,13 @@ locals {
       resources  = ["deployments", "configmaps", "pods", "secrets", "services", "endpoints"]
     }
   ]
+
+  # define Kubernetes policy for deployer: To Be Refined later
+  k8s_deployer_access = [
```

Comment:
This isn't enough access to be able to deploy.

Reply:
make all * for the time being.

Reply:
set same as operator